DependencyInjection/Compiler/EntityListenerPass.php

```diff
@@ -39,20 +39,50 @@ public function process(ContainerBuilder $container)
                     continue;
                 }
 
-                $resolver = sprintf('doctrine.orm.%s_entity_listener_resolver', $name);
-                if ($container->hasAlias($resolver)) {
-                    $resolver = (string) $container->getAlias($resolver);
+                $resolverId = sprintf('doctrine.orm.%s_entity_listener_resolver', $name);
+                if ($container->hasAlias($resolverId)) {
+                    $resolverId = (string) $container->getAlias($resolverId);
                 }
 
-                if (!$container->hasDefinition($resolver)) {
+                if (!$container->hasDefinition($resolverId)) {
                     continue;
                 }
 
+                $resolver = $container->getDefinition($resolverId);
```

Comment:
all this could be simplified using ``$resolver = $container->findDefinition($resolverId);`` to let the container resolve aliases by itself instead of doing it manually (and using ``!$container->has($resolverId)`` in the continue block, to account for aliases)